### DISCUSSION TITLE
Report Scammer- https://github.com/miracle1125

### DISCUSSION THREAD

**Discussion by happyeagle912** [community/community#121684] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

This is scammer's github. 
please consider this and be careful to be avoid him.

**Comment by Foteve** [comment]
https://github.com/orgs/community/discussions/121684#discussion-6587334

**Comment by Foteve** [comment]
Video movies

**Comment by Foteve** [comment]
Mobali hack

**Comment by queenofcorgis** [comment, marked as answer]
Hi @happyeagle912 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
